### PR TITLE
Add some snippets for the OffsetDate type

### PR DESCRIPTION
**Pull request by carlosschults**

### CODE REVIEW

File: src/NodaTime.Demo/OffsetDateDemo.cs [hunk 1]
```diff
@@ -0,0 +1,61 @@
+﻿// Copyright 2010 The Noda Time Authors. All rights reserved.
```
Comment (jskeet) [inline src/NodaTime.Demo/OffsetDateDemo.cs]: Please change to 2019. (I keep doing this too...)

**Comment by jskeet** [comment]
Why have you closed this, out of interest? Was that an accident?

**Comment by carlosschults** [comment]
@jskeet I've closed it due to the build failing, yes. Thought something went wrong.

**Comment by jskeet** [comment]
Nope, let's reopen and I'll try to work out what's wrong.

**Comment by carlosschults** [comment]
Sounds good to me.

**Comment by jskeet** [comment]
Hmm... I note that the YearMonth snippets haven't appeared, either. Will look into whatever's failed in terms of the build on Bagpuss (which builds the docs).

**Comment by jskeet** [comment]
Okay, I see what's wrong with the code:

- There's no LocalDate property, it's Date
- The import should be for NUnit.Framework, not Nunit.Framework
- You don't have a System import

I'm surprised this was compiling locally for you...

**Comment by jskeet** [comment]
(Have fixed those in another commit - let's see what happens now...)

**Comment by jskeet** [comment]
Okay, I've worked out what was wrong with the YearMonth build, and fixed up the tests in the OffsetDate demo too (and the copyright). Fingers crossed this will now go green, at which point I can merge.

**Pull request merged**